jenkins-scripts/dsl/_configs_/Globals.groovy

```diff
@@ -102,11 +106,11 @@ class Globals
 
    static ArrayList get_ros2_suported_distros()
    {
-     return [ 'dashing', 'eloquent' ]
+     return [ 'dashing', 'eloquent', 'foxy', 'rolling' ]
    }
 
    static String get_ros2_development_distro() {
-     return 'foxy'
+     return 'galactic'
```

Comment:
Should this be 'rolling'?

Reply:
Yeah maybe this entire logic could be a bit updated to account for the new rolling paradigm. The reason I didn't put `rolling` here is that we probably want rolling install jobs. See


https://github.com/ignition-tooling/release-tools/blob/c5045684ef7915bc29c6573ad4a624de5b402084/jenkins-scripts/dsl/gazebo_ros_pkgs.dsl#L165-L168

Reply:
Yes, it probably makes sense to have install jobs.  I think now generally that the ordering may be a bit inverted, in that we will have packages for rolling, that will eventually become the packages for galactic and further.

Reply:
Cool, so I think `galactic` here is ok for now?

Reply:
agree with Louise. The logic probably needs to be updated to match what currently is the "developement_distro".

Reply:
:+1: